Winston not being found when loading in Roosevelt

When running `npm install` on a new Roosevelt app that includes the changes for logger, it will fail when running the postInstall scripts due to the config auditor needing it. @mulligan121 has also reproduced this bug.

**Steps to Reproduce**
1) Setup simple roosevelt app with `yo roosevelt`
2) Open the package.json and switch the npm dependency for roosevelt to be the master branch of roosevelt on github:
```json
"roosevelt": "rooseveltframework/roosevelt",
```
3) run `npm install` and see the error.

```
> node ./lib/scripts/configAuditor.js

module.js:549
    throw err;
    ^

Error: Cannot find module 'winston'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/benjaminlannon/Development/roosevelt-client-exposed-views-test-app/node_modules/roosevelt/lib/tools/logger.js:2:17)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
```

**Solution**

I'll move `winston` to an actual dependency rather than a dev dependency as it currently is.

--- a/package.json
+++ b/package.json
@@ -38,7 +38,8 @@
     "tamper": "~1.0.0",
     "tmp": "~0.0.33",
     "toobusy-js": "~0.5.0",
-    "vnu-jar": "~18.8.0"
+    "vnu-jar": "~18.8.0",
+    "winston": "3.1.0"
   },
   "devDependencies": {
     "codecov": "3.1.0",
@@ -54,8 +55,7 @@
     "sinon": "6.3.4",
     "standard": "12.0.1",
     "supertest": "3.3.0",
-    "teddy": "0.4.26",
-    "winston": "3.1.0"
+    "teddy": "0.4.26"
   },
   "repository": {
     "type": "git",

